content/installation/deploying-a-manager-image.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: bt_wiki
+title: Deploying a Manager Image
+category: Installation
+draft: false
+weight: 601
+
+---
+
+{{% gsNote title="Prerequisites" %}}
+ * A manager image built using the [Image Builder]({{< relref "image-builder.md" >}})
+ * For the command-line install, the [`cfy` command]({{< relref "/cli/overview" >}})
+ * Account credentials for the platform you are deploying on
+{{% /gsNote %}}
+
+
+ 1. Create an instance of the [image builder]({{< relref "image-builder.md" >}}) image
+ 2. Make a note of the IP/hostname
+
+    ```bash
+    $ CLOUDIFY_HOST={your-manager-ip}
+    ```
+
+
+# Command Line
+
+ 1. Swith to the new manager:
+
+    ```bash
+    $ export CLOUDIFY_USERNAME=cloudify
+    $ export CLOUDIFY_PASSWORD=cloudify
+    $ export CLOUDIFY_SSL_TRUST_ALL=true
+    $ cfy use --port 443 --management-ip ${CLOUDIFY_HOST}
+    ```
+
+    {{% gsNote title="TLS" %}}
+    If you have disabled security then use port 80 instead (in this case the `export` lines above are not required): `$ cfy use --port 80 --management-ip ${CLOUDIFY_HOST}`.
+
+    {{% /gsNote %}}
+
+ 2. Create an input file for the `CloudifySettings` blueprint
+    ```bash
+    $ echo >install.yaml <<EOF
+    {
+      "aws_access_key": "{your access key}",
+      "aws_secret_key": "{your secret key}",
+      "agents_security_group_name": "{your desired group}",
+      "user_ssh_key": "{your ssh key}",
+      "agents_user": "centos",
+      "agents_keypair_name": "{your desired keypair name}",
+      "new_manager_username": "cloudify",
+      "new_broker_password": "a bad password",
+      "broker_names_and_ips": "localhost",
+      "new_broker_username": "cloudify",
+      "manager_names_and_ips": "${CLOUDIFY_HOST}",
+      "new_manager_password": "a really bad password"
+    }
+    EOF
+    ```
+ 3. Create the deployment
+    ```bash
+    $ cfy deployments create --deployment-id install --blueprint-id CloudifySettings --inputs install.yaml
+    ```
+ 4. Start the `install` workflow
+    ```bash
+    $ cfy executions start --deployment-id install --workflow install
+    ```
+
+ 5. Save the TLS certificate for use with `cfy` in future
+
+    ```bash
+    # Stop trusting all:
+    $ CLOUDIFY_SSL_TRUST_ALL=
+
+    $ export CLOUDIFY_SSL_CERT=${CLOUDIFY_HOST}.cert
+    $ echo | openssl s_client -connect 54.164.100.144:443 2>/dev/null | sed -n '/^-----BEGIN CERTIFICATE-----$/,/^-----END CERTIFICATE-----$/p' >${CLOUDIFY_SSL_CERT}
```

Comment:
Might be good to comment this line to explain what it does, and replace the IP with a variable